test/jbuilder/schema/builder_test.rb

```diff
@@ -54,20 +54,14 @@ class Jbuilder::Schema::BuilderTest < ActiveSupport::TestCase
 
   test "renders a template with view assigns" do
     Dir.chdir("./test/fixtures") do
-      schema = Jbuilder::Schema.render template: "articles/index", assigns: { articles: Article.all }, title: "Article", description: "Article in the blog"
+      schema = Jbuilder::Schema.render template: "articles/index", assigns: { articles: Article.all }
 
       assert_equal({
-        type: :object,
-        title: "Article",
-        description: "Article in the blog",
-        required: [],
-        properties: {
-          "articles" => {
-            type: :array,
-            items: {
-              "id" => {type: :integer},
-              "title" => {type: :string}
-            }
+        "articles" => {
+          type: :array,
+          items: {
+            "id" => {type: :integer},
+            "title" => {type: :string}
```

Comment:
@newstler do you happen to know which is the more correct structure here? The template is just `json.articles @articles, :id, :title` and I think the new test is more accurate.

I'll merge for now, and then I can revisit.